delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutesSpec.scala

```diff
@@ -219,6 +203,39 @@ class BlazegraphViewsRoutesSpec
       }
     }
 
+    "fail to fetch statistics and offset from view without resources/read permission" in {
```

Comment:
I moved the tests up as the update on the view introduce a restriction on the tag